convergedtarkus/gosymlink
# #22 Use go1.11

## Problem

Go 1.10 is pretty old (heck go 1.12 is out!), probably should upgrade to something newer.

## Solution

Use go 1.11.6 (the latest go 1.11 version). Update CI as well.
Also added misspell to help reduce misspellings in the program, and guess what, there are already a bunch...

## Implementation Notes

- Use go 1.11.6
- Use misspell (https://github.com/client9/misspell) to verify spelling on CI

## Tests Added

None yet, see #10 

## Testing Instructions

CI passes
Ensure `make installDevTools` installs misspell and the new `make checkSpelling` (and `make checkAll`) works correctly.
